### ISSUE TITLE
Screen Reader Compatibility

### ISSUE THREAD

**Issue by Relurk1** [Relurk1/data-visualizer#1]
Most images lack the required alt text for screen readers. Adding this and verifying that the rest of the site is screen reader compatible would go a long way to ensuring that maximum accessibility
